src/Core/AndroidNative/maui/src/main/java/com/microsoft/maui/ContextHelper.java

```diff
@@ -0,0 +1,20 @@
+package com.microsoft.maui;
+
+import android.content.res.TypedArray;
+import android.content.Context;
+import java.lang.reflect.Field;
+
+public class ContextHelper {
+    public static final TypedArray getStyledAttributesForStyleable(Context context, String styleable, int resId) {
+        try {
+            Field field = R.styleable.class.getField(styleable);
+            int [] attributes = (int[]) field.get(R.styleable.class);
```

Comment:
Does this line crash with Java `NoSuchFieldException`? I feel like I hit this problem once and passed in the C# `int` instead, but I didn't dig into how to fix this...

Reply:
So I figured out why we don't have a `R.styleable`. Its because we are not generating an `R.txt` file for this particular file (maui.aar). The reason being that is does not contain any of its own resources. 

The XA `ResolveLibraryProjectImports` skips this .aar because it does not contain `res` folder. As a result we never add it to the `@(LibraryResourceDirectories)` ItemGroup. This is the source item group which controls the generation of the `R.java` files by `GenerateLibraryResources`. But it is also the group that is used by `aapt2` eetc. 

So to get this code to work we would need to make sure we include the `maui.aar` file in those ItemGroups so we end up with a `R.java`. We could possibly, create a new group specifically for the `R.java` generation, so that all libraries get one. But that will impact on apk size since it will end up generating code which is not used.

Reply:
I worked around this by adding a single "unused" resource to the `maui.aar` package. This will force Xamarin.Android to generate an `R.java` for this package. It then allows us to access the resources we need :D